src/components/ContextMenu.scss

```diff
@@ -73,4 +73,15 @@
   .context-menu-option:focus {
     z-index: 1;
   }
+
+  @media screen and (max-width: 1000px) {
+    .context-menu-option {
+      display: block;
+      margin-inline-end: 0px;
+
+      .context-menu-option__shortcut {
+        visibility: hidden;
```

Comment:
``visibility: hidden`` only hide the element but leaves a blank gap.

try:  ``display: none``